src/styles/annotator/highlights.scss

```diff
@@ -33,6 +54,52 @@
   }
 }
 
+// Configure clustered highlight styling. The `.hypothesis-highlights-clustered`
+// class is managed by `highlight-clusters`
+.hypothesis-highlights-clustered .hypothesis-highlight {
+  // When clustered highlights are active, use an opaque blue for focused
+  // annotations so don't end up with a funny color mix
+  --highlight-color-focused: var(--hypothesis-color-blue);
+
+  &.user-annotations {
+    --highlight-color: var(--hypothesis-user-annotations-color);
+    --highlight-decoration: var(--hypothesis-user-annotations-decoration);
+
+    & > .user-annotations {
+      --highlight-color: var(--hypothesis-user-annotations-color);
+      --highlight-blend-mode: var(--hypothesis-cluster-blend-mode);
+    }
+  }
+
+  &.user-highlights {
+    --highlight-color: var(--hypothesis-user-highlights-color);
+    --highlight-decoration: var(--hypothesis-user-highlights-decoration);
+
+    & > .user-highlights {
+      --highlight-color: var(--hypothesis-user-highlights-color);
+      --highlight-blend-mode: var(--hypothesis-cluster-blend-mode);
+    }
+  }
+
+  &.other-content {
+    --highlight-color: var(--hypothesis-other-content-color);
+    --highlight-decoration: var(--hypothesis-other-content-decoration);
+
+    & > .other-content {
+      --highlight-color: var(--hypothesis-other-content-color);
+      --highlight-blend-mode: var(--hypothesis-cluster-blend-mode);
+    }
+  }
+}
+
+// No matter what kind of highlight styling is applied, make sure nested
+// highlights don't pile up too high with blending.
+.hypothesis-highlight {
+  & & & & & {
```

Comment:
Let's stick a pin in this. I chose this depth because with the `multiply` blending and the amount of colors, it "felt" right when I was working with deeply-nested, multi-cluster highlights (i.e. it's not arbitrary). 

There's gonna be a deep dive next week on the details of highlight coloring and blending, so we will certainly revisit.